mozilla/bedrock
# Move VPN Resource Center images from Contentful CDN to Bedrock media

_Do not merge for now_.

## One-line summary

This changeset moves the VPN RC away from using images live-hosted by Contentful. 

It:
* adds all of the images, in the appropriate size, from the Contentful CDN to our bedrock codebase
* contains a data migration that, when run, updates the `contentful_contentfulentry` table appropriately. (It swaps the `src` and/or `srcset` attributes of `img` tags from full Contentful CDN URLs to local image paths instead, matching the files added to the codebase. 
* EDIT: stops the regular syncing of Contentful content, else a change (or a forced update on Dev or a Demo) would overwrite the changes to use the local files

## Significant changes and points to review

The data migration is not pretty, but should make sense. Please call out anything that looks too strange. 

## Issue / Bugzilla link

No ticket

## Testing
Locally:

* on `main` run `./bin/run-db-download.py --force`
* switch to this branch 
* `./manage.py migrate` which should apply one migration (0007) in the contentful app
* Go to http://localhost:8000/en-US/products/vpn/resource-center/
   * Confirm there are no missing images, broken HTML, etc and that images are loaded from localhost not https://images.ctfassets.net
   * View each child page and confirm all OK and loading local images - having the Network tab in Devtools helps
   * When a child page contains an image, use devtools to drop the DPR from 2 to 1  and confirm via the Network tab that the lower-res image is loaded, and that it's locally sourced
